### ISSUE TITLE
Whoops not propagating to exception handler for certain errors on Windows

### ISSUE THREAD

**Issue by jakezatecky** [filp/whoops#239]
I've been trying to figure this out for the past few hours. Consider the following code that should produce a Whoops exception trace:

``` php
$whoops = new \Whoops\Run;
$whoops->pushHandler(new \Whoops\Handler\PrettyPageHandler);
$whoops->register();

$j->invalidMethod();  // Undefined variable
```

This error is surfaced by Whoops on my CentOS machine. However, on my Windows machine, this error is not surfaced, and the native PHP error handler will give me a stack trace, not Whoops.

It's very important to note that if I use a defined variable, the error does get captured on the Windows machine:

``` php
$i = "";
$i->invalidMethod();  // Will surface to Whoops
```

I've tested this with PHP 5.3 and 5.4 on both Windows and CentOS. All machines have `error_reporting = E_ALL` in `php.ini`. This inconsistency appears to be purely isolated to any Windows machines I use.

I think this might be related to the fact that really two errors are going on here: an undefined variable, and a method invocation on a non-object. If I force `Whoops\Run::canThrowExceptions` to `false` in all scenarios, then the exception will be propagated on my Windows machine.

Any thoughts on how to surface this error? I'm using Laravel and it will suppress any native PHP stack traces in favor of Whoops, which leads to the "white screen of death" in this scenario (and some others), which forces me to go into the PHP log (and not the Laravel log), which means debugging painful.

**Comment by GRMule** [comment]
I tried this (Windows 7 + PHP 5.5.12) and could not reproduce your results. The first example surfaced as a Whoops exception, "ErrorException (E_NOTICE) Undefined variable: t".

The latter one gave me Whoops output with ""ErrorException (E_ERROR) Call to a member function invalidMethod() on a non-object". I see I do have xDebug output for this one behind the Whoops output, but that's probably down to how I implemented the shutdown handler to catch fatals.

Hard to chalk it up to a PHP version difference, I don't see anything that's grabbing my attention here: http://3v4l.org/5FXob or here: http://3v4l.org/IR7HJ -- in both cases, the 5.X output was consistent. A Windows-specific error is relatively unusual, too.
